conf/citest/pipelines/LoadMembers.json

```diff
@@ -0,0 +1,73 @@
+{
+    "reference_db": "mysql://ensro@mysql-ens-compara-prod-6:4616/dthybert_citest_load_members_99",
+    "database_tests": {
+        "dnafrag": [
+            {   "test": "num_rows"
+            },
+            {   "test": "content" }
+        ],
+        "gene_member": [
+            {   "test": "num_rows"
+            },
+            {   "test": "content",
+                "args": {
+                    "ignore_columns": "canonical_member_id"
+                }
+            }
+        ],
+        "exon_boundaries": [
+            {   "test": "num_rows"
+            },
+            {   "test": "content",
+                "args": {
+                    "ignore_columns": ["gene_member_id", "seq_member_id"]
```

Comment:
So the idea of using a `-` at the start of column names has been dropped in favour of this `ignore_columns` arg?

Reply:
Yes, because MySQL allows column names to start with a `-`, so although I know we don't name columns that way, I have thought it would be safer to handle the inclusion/exclusion list this way (and may be simpler for users too).